sig-openstack/README.md

```diff
@@ -1,7 +1,10 @@
 # OpenStack SIG
 
-This is the wiki page of the Kubernetes OpenStack SIG: a special interest group
-co-ordinating contributions of OpenStack-related changes to Kubernetes.
+This is the community page of the Kubernetes OpenStack SIG: a special
+interest group coordinating the cross-community efforts of the OpenStack
+and Kubernetes communities. This includes OpenStack-related contributions
+to Kubernetes projects, with OpenStack as a deployment platform, a
+service provider, or applications to run on Kubernetes.
```

Comment:
By "applications to run on Kubernetes" here OpenStack as "a complex application to run on Kubernetes"? Something about the flow is a little unclear to me, might also work better as a bullet list for this last piece - otherwise looks good :).